src/Component/Symfony/Request/Factory/RequestConfigurationFactory.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Resource\Symfony\Request\Factory;
```

Comment:
I'm wondering what is our go-to architecture structure here. Main questions that I have right now in my head: 
* Should these classes be in component or in bundle namespace?
* Shouldn't we earlier combine component and bundle namespace?
* Once the component and bundle namespaces are merged, then we can refactor the structure to have Sylius\ResourceBundle namespace and there we could have Symfony subfolder perhaps

Reply:
Yes, I'll vote for Sylius\Resource namespace instead of Sylius\ResourceBundle.

I need to check if it's necessary to do this right now. Cause I really don't remember why it was needed now, cause I'm trying not to use RequestConfiguration anymore.

Postpone this one, the other one in review is a most important one.